src/models/ministryPlate.ts

```diff
@@ -123,12 +139,12 @@ export class MinistryPlateDocument extends DocumentModel {
     this.Reissue = { Reason: plate.plateReasonForIssue };
 
     // S3 metadata
-    this.metaData.vrm = techRecord.primaryVrm ?? techRecord.trailerId;
+    this.metaData.vrm = techRecord.techRecord_vehicleType === 'hgv' ? techRecord.primaryVrm : techRecord.trailerId;
```

Comment:
Thin you need to invert the ternary here as well:
```suggestion
    this.metaData.vrm = techRecord.techRecord_vehicleType === 'trl' ? techRecord.trailerId : techRecord.primaryVrm;
```

Reply:
With this being plates I'm not sure it does right? I think they're only applicable to HGV or TRL

Reply:
Yes, this is written badly but I'm not in the mood or have the time to improve it!